static/js/topic_page/ranking_tile.tsx

```diff
@@ -126,15 +139,15 @@ function fetchData(
         for (const place in statData[item.statVar].stat) {
           const rankingPoint = {
             placeDcid: place,
-            stat: statData[item.statVar].stat[place].value,
+            value: statData[item.statVar].stat[place].value,
           };
-          if (rankingPoint.stat === undefined) {
+          if (typeof rankingPoint.value === "undefined") {
```

Comment:
should be sufficient to test `rankingPoint.value === undefined` rather than using typeof

Reply:
I could not pass the code style check with `rankingPoint.value === undefined`. It told me to avoid using undefined because it can be overwritten and it is not a keyword, so I change to use typeof...

Reply:
maybe we can use the lodash method _.isUndefined()

Reply:
Done